packages/story-editor/src/components/canvas/useInsertElement.js

```diff
@@ -140,11 +59,7 @@ function useInsertElement() {
       const element = createElementForCanvas(type, props);
       const { id, resource } = element;
       addElement({ element });
-      if (resource) {
-        postProcessingResource(resource);
-        handleRegisterUsage(resource);
-        generateBaseColor(element);
```

Comment:
If `baseColor` is missing from some previous local media, will it still be added somewhere?

Reply:
Yes, `postProcessingResource` is invoked in a few other places, mainly in the media library panel whenever items change ther.e

Reply:
What about hotlinked images? Or images inserted for WP media library?